hack/cluster-migration/main.go

```diff
@@ -403,6 +403,10 @@ func checkIfEligible(job cfg.JobBase) (bool, string) {
 		return false, reason
 	}
 
+	if job.Spec.ServiceAccountName != "" {
```

Comment:
The field `serviceAccountName: prowjob-default-sa` make the job eligible for migration